packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -477,16 +477,11 @@ type OptionalType = (Parameter | Property) & {
   type: { optional: true };
 };
 
-function isOptional(
-  param: Parameter | Property
-): param is OptionalType {
+function isOptional(param: Parameter | Property): param is OptionalType {
```

Comment:
Is this because different prettier length options ? I've seen that kind of changes a lot.